common/setup.py

```diff
@@ -1,13 +1,25 @@
 import os
 from setuptools import find_packages, setup
+import re
 
 # allow setup.py to be run from any path
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
 
+# code found at: https://stackoverflow.com/a/7071358/6095378
+VERSIONFILE = "data_refinery_common/_version.py"
+verstrline = open(VERSIONFILE, "rt").read()
```

Comment:
Do you mind using underscore to make `verstrline`, `VSRE` and `verstr` more readable?